src/templates/taskresult/view/controlbuttons/controlbuttons.html

```diff
@@ -1,4 +1,4 @@
-<div class="col-md-12 form-group odalic-hr-pushbtm">
+<div class="row form-group odalic-hr-pushbtm">
     <!-- Buttons on the left -->
     <div class="col-md-2">
         <button class="btn btn-lg btn-block" ng-click="state = state - 1" ng-disabled="state <= 0">
```

Comment:
Resolve the conflict with develop, first. Or: is this even needed? Did @istvan-satmari solve this already, @KataBoku ? Thanks!